notebooks/fixtures/config-exp/user.yaml

```
@@ -0,0 +1,5 @@
+user:
+  ACCOUNT: zrtrr
+  MACHINE: hera
```

You could either make the other two changes I suggested to substitute `{{ user.PARMdir }}` with `/path/to/ufs-srweather-app/parm`, or add an entry here. Either way, the there wouldn't be an un-rendered Jinja expression remaining in those entries. Choosing this option is DRYer.

```suggestion
  MACHINE: hera
  PARMdir: /path/to/ufs-srweather-app/parm
```

---

Thanks, I like this DRY version.